package.json

```diff
@@ -38,14 +38,14 @@
     "@babel/runtime": "^7.14.0",
     "@open-web3/dev-config": "^0.1.13",
     "@open-web3/util": "^1.0.2-0",
-    "@polkadot/api": "^9.11.2",
-    "@polkadot/api-augment": "^9.11.2",
-    "@polkadot/api-derive": "^9.11.2",
+    "@polkadot/api": "^9.12.1",
```

Comment:
The latest version of Polkadot.js API is "^9.13.6". Could you use the latest packages?